XenAdmin/Wizards/NewVMWizard/Page_Storage.cs

```diff
@@ -409,7 +409,7 @@ private void EditButton_Click(object sender, EventArgs e)
                 if (dialog.ShowDialog(ParentForm) != DialogResult.OK)
                     return;
 
-                selectedItem.Disk = dialog.NewDisk();

```

Comment:
Although I could not reproducde this, I suspect that what happened is that the SrPicker.SR was null when NewDisk() was called here, hence the XenRef<SR> constructor within this method crashed,hence I changed where the disk is created in the NewDiskDialog.